components/cards/Cards.module.css

```diff
@@ -0,0 +1,52 @@
+.recipecard {
+  border: 3px solid var(--MainClr);
+  border-radius: 15px;
+  padding: 5px;
+  width: 80vw;
```

Comment:
I wouldn't use these absolute values here, just use the `vw` and `vh` properties very carefully